repository_service_tuf/helpers/tuf.py

```diff
@@ -140,9 +140,7 @@ def _update_timestamp(snapshot_version: int):
         _sign(timestamp, Timestamp.type)
         _add_payload(timestamp, Timestamp.type)
 
-    def _update_snapshot(
-        targets_meta: List[Tuple[str, int]]
-    ) -> Metadata[Snapshot]:
+    def _update_snapshot(targets_meta: List[Tuple[str, int]]) -> int:
```

Comment:
I am honestly not sure why here we used `Metadata[Snapshot]` as a return type when we do return an int - the new snapshot version.

Reply:
Yeah, I think it was just a stray mistake, perhaps influenced by the fact that the `_load` method returns a `Metadata` type.